src/index.ts

```diff
@@ -0,0 +1,124 @@
+class School {
+  directions: Direction[] = [];
+
+  addDirection(direction: Direction) {
```

Comment:
Please, do not forget to add explicitly type annotation for the function return type.